lessons/5-write-and-use-files.md

```diff
@@ -33,14 +35,67 @@ The user confirms the input by pressing the enter key. Since the enter key also
 
 Keep in mind that `gets` returns a string. ;)
 
-### Homework
+#### Homework
+
+Write an interactive ruby program that processes user input (e.g. via the gets method). It should live in its own file (e.g. homework.rb) and can be executed like `ruby homework.rb`. (e.g. in repl or in our container)
 
-Write an interactive ruby program that processes user input (e.g. via the gets method). It should live in its own file (e.g. homework.rb) and can be executed like `ruby homework.rb` inside our container.
 Bonus 1:
-Use method(s)
+Use one ore more methods
+
 Bonus 2:
 Use loops and allow the user to decide when to exit the program
 
 If you're lacking inspiration try to write a small addition trainer. ➕
+Two inputs and the solution and your program tells you if the input is correct.
+
+[Possible solution](/lessons/examples/lesson-5-calc.rb)
+
+# Deutsch
+
+## Lesson 5 - Schreiben und benutzen von Dateien
+
+### Erstellen und ausführen von Dateien
+
+1. Gehe zu deinem Ordner welchen du mit deinem Container synchronisiert hast und erstelle eine neue Datei. Den Namen der Datei kannst du freiwählen, aber die Endung muss `.rb`. `.rb` is hier die Abkürzung für Ruby und ist wichtig für den Interpreter.
+
+2. Wir haben folgende Datei erstellt: [lesson-5.rb](/lessons/examples/lesson-5.rb)
+
+3. Nun öffne die Datei mit dem Editor [Atom](https://atom.io). :)
+   Wir haben die folgende Zeile hinzugefügt:
+   ```ruby
+   puts "Hello World"
+   ```
+
+4. Nun starte deinen Container und führe den folgenden Befehl aus:
+   ```shell
+   ruby lesson-5.rb
+   ```
+   
```

Comment:
What do you think about adding a hint here that `lesson-5.rb` is merely the name of the file created in earlier steps and that a different filename can be correct here if the previously created file has a different name?